ut/coverage-not-yet

```diff
@@ -0,0 +1,7 @@
+include/call_list_store_processor.h
```

Comment:
This is a lot of uncovered files! What kind of coverage is there?

Reply:
Coverage is pretty poor in this area, although no poorer as a result of these changes. Worth me adding some UTs?
```
------------------------------------------------------------------------------
                           GCC Code Coverage Report
Directory: /home/ubuntu/sprout/plugins/memento-as
------------------------------------------------------------------------------
File                                       Lines    Exec  Cover   Missing
------------------------------------------------------------------------------
src/mementoappserver.cpp                     195     195   100%
src/httpnotifier.cpp                          31      16    51%   48,51,54,57-61,63-65,67-68,73-74
modules/memento-common/src/mementosaslogger.cpp
                                               3       1    33%   14,19
src/call_list_store_processor.cpp            101      16    15%   51,63,65-71,73-74,77,81-85,88,91-92,99,101,104-105,107,111,113,115,117,121,123,125,129,131-132,137-138,142-143,145-146,149-150,156,167-168,171-172,174,181,186,190,195,197,199,202,207,209-210,212,215,217,220-221,223,229-231,234-235,237,244,246,248,250,253,255-259,261,267-268,271
modules/memento-common/src/call_list_store.cpp
                                             167       7     4%   42,44,47,50,53,73,76,78,80,82,84,86,88,98,101,107-111,117,119,129,137-138,140-141,143,147,149,152-157,165-169,172-173,176-177,183,186-187,190,193,197,199-200,204-205,208,213,220-222,224-225,227,231,234-236,240,246,248-249,255-256,258,266-268,270,272,275-279,281,283,285,288-292,295,298,302,304-305,309-310,312,314-315,318,320,327,333-334,336-337,339,342,344,347-350,358-360,363-371,374,376,379-380,383,386,390,392-393,397-398,401,405,414,423,425-426,428-429,434,438,440,442,445,447-448,453,460-462,464-465
modules/memento-common/include/call_list_store.h
                                               1       0     0%   21
include/call_list_store_processor.h            3       0     0%   57,68,72
------------------------------------------------------------------------------
TOTAL                                        501     235    46%
------------------------------------------------------------------------------
```